ee/localserver/jwk_keys.go

```diff
@@ -0,0 +1,89 @@
+package localserver
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"math/big"
+)
+
+type jwk struct {
+	Type  string `json:"kty"`
+	Curve string `json:"crv"`
+	X     string `json:"x"`
+	Y     string `json:"y"`
+	KeyID string `json:"kid"`
+}
+
+// jwkToECDSAPublicKey converts a JWK JSON string into an ECDSA public key.
+func (jwk jwk) ecdsaPubKey() (*ecdsa.PublicKey, error) {
+	// Ensure the key type is EC.
+	if jwk.Type != "EC" {
```

Comment:
For documentation, I think it would be helpful to add a link to https://datatracker.ietf.org/doc/html/rfc7518#section-6.2, and note that the jwk.Type and jwk.Curve values are case-sensitive. Could maybe move `"EC"`, `"P-256"`, `"P-384"`, `"P-521"` to consts so we can document all that in one place?